edx_proctoring/backends/rest.py

```diff
@@ -8,9 +8,12 @@
 import uuid
 import jwt
 
+from six.moves.urllib.parse import urlparse
 from webpack_loader.utils import get_files
 from webpack_loader.exceptions import BaseWebpackLoaderException, WebpackBundleLookupError
 
+from django.conf import settings
```

Comment:
I think I got this import order right? 

https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#imports